core/templates/dev/head/services/AssetsBackendApiService.ts

```diff
@@ -63,19 +61,17 @@ oppia.factory('AssetsBackendApiService', [
       }
 
       $http.get('/appidentityhandler').then(function(response) {
-        if (!DEV_MODE && !response.data.GCS_RESOURCE_BUCKET_NAME) {
+        if (!response.data.GCS_RESOURCE_BUCKET_NAME) {
           throw Error('GCS_RESOURCE_BUCKET_NAME is not set in prod.');
         }
```

Comment:
Now that I think of it. Remove this `if` altogether.

Reply:
I don't think this if should be removed since it validated the result gotten from the handler